### ISSUE TITLE
Vue: Add any applicable Vue 3 bug fixes to Vue 2

### ISSUE THREAD

**Issue by emccorson** [OnsenUI/OnsenUI#2903]
Some of the bug fixes for Vue 3 will also work to fix bugs in Vue 2.

Go through the CHANGELOG for Vue 3 and apply any fixes that are applicable to Vue 2.

Should be done after #2902.

**Comment by emccorson** [comment]
No longer relevant.
